In [1]:
import pandas as pd
import numpy as np

In [2]:
data = pd.read_csv('house_price_Dống-Da_Hà-Nội_subdata.csv')

In [3]:
data.head()

area    price  postDate land_certificate house_direction balcony_direction  \
0  50.0  14700.0     44197            So do             NaN               NaN   
1  57.0   4200.0     44265            So do             Nam               Nam   
2  62.0  11000.0     44268            So do             NaN          dong-Nam   
3  26.0   2900.0     44292            So do             NaN               NaN   
4  40.0   3950.0     44292            So do             NaN               NaN   

   toilet  bedroom  floor        lat        long property_type  
0     4.0      6.0    6.0  21.015698  105.824583     trong ngo  
1     3.0      8.0    4.0  21.025208  105.833111     trong ngo  
2     6.0      5.0    6.0  21.025379  105.801011     trong ngo  
3     4.0      3.0    4.0  21.014325  105.834918     trong ngo  
4     NaN      NaN    5.0  21.009862  105.838083     trong ngo

In [4]:
data.describe()

area         price      postDate      toilet    bedroom  \
count    665.000000  6.170000e+02    671.000000  240.000000  320.00000   
mean     103.763534  2.892758e+04  44259.903130    3.645833    4.31875   
std      471.861867  1.464474e+05     58.280392    2.349126    2.58976   
min        0.000000  4.500000e+00  44197.000000    1.000000    1.00000   
25%       39.000000  3.950000e+03  44197.000000    2.000000    3.00000   
50%       55.000000  8.900000e+03  44292.000000    4.000000    4.00000   
75%       80.000000  2.150000e+04  44312.000000    4.250000    6.00000   
max    11000.000000  3.300000e+06  44325.000000   16.000000   22.00000   

            floor         lat        long  
count  366.000000  671.000000  671.000000  
mean     4.762295   21.015733  105.825550  
std      1.676853    0.006517    0.009011  
min      1.000000   20.998499  105.795544  
25%      4.000000   21.011094  105.822605  
50%      5.000000   21.018072  105.828985  
75%      5.000000   21.018777  105.829949  
max     11.000000   21.036845  105.841407

In [5]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 671 entries, 0 to 670
Data columns (total 12 columns):
 #   Column             Non-Null Count  Dtype  
---  ------             --------------  -----  
 0   area               665 non-null    float64
 1   price              617 non-null    float64
 2   postDate           671 non-null    int64  
 3   land_certificate   369 non-null    object 
 4   house_direction    42 non-null     object 
 5   balcony_direction  24 non-null     object 
 6   toilet             240 non-null    float64
 7   bedroom            320 non-null    float64
 8   floor              366 non-null    float64
 9   lat                671 non-null    float64
 10  long               671 non-null    float64
 11  property_type      671 non-null    object 
dtypes: float64(7), int64(1), object(4)
memory usage: 63.0+ KB


Bộ dữ liệu có 12 cột và 671 dòng:

    * cột area,price,toilet, bedroom, floor thuốc kiểu dữ liệu float, mang tính chất định lượng => thước đo ratio
    * cột lat,long cũng mang kiểu dữ liệu float64 tuy nhiên nó dùng để diễn tả vị trí không thể phân tích dựa vào thước đo
    * cột postDate, đại diện cho ngày đăng tin nhưng mang kiểu dữ liệu int64
    * cột land_certificate, house_direction, balcony_direction, house_direction, property_type dược lưu trữ bởi kiểu dữ liệu object mang tính chất định danh=>thước đo normal

In [ ]:
data1 = data[data['']]